frontend/src/store.js

```diff
@@ -597,6 +597,25 @@ const builder_or_trainer = {
   }
 }
 
+const clipboard = {
+  state: {
+    clipboard_data: undefined,
+  },
+  getters:{
+    get_clipboard: state => {
+      return state.clipboard_data;
+    }
+  },
+  mutations: {
+    set_clipboard(state, data) {
+      state.clipboard_data = data
+    },
+    clear_clipboard(state, data) {
```

Comment:
do we want to call this on logout or somewhere? https://github.com/diffgram/diffgram/blob/479cfb02a07b483c477a1cd993f18b216f4b2f63/frontend/src/store.js#L97

maybe clear sets back to undefined and doesn't require data?

Reply:
Good idea! Added it here: https://github.com/diffgram/diffgram/pull/243/commits/90c325905c9d8fcadb81ed113691826b27bf6d9b